<?xml version="1.0" encoding="UTF-8"?>
<launch>
	
	<arg name="latitude" default="60.29499" />
	<arg name="longitude" default="5.36145" />

	<node name="tb_simulate_actuatorresponse_node"  pkg="tb_simulate" type="tb_simulate_actuatorresponse_node" 	respawn="true"/>
	<node name="tb_simulate_djiresponse_node"    		pkg="tb_simulate" type="tb_simulate_djiresponse_node" 				respawn="true">
	  <param name="reference_latitude" value="$(arg latitude)"/>
	  <param name="reference_longitude" value="$(arg longitude)"/>
	  <param name="reference_altitude" value="0"/>
	</node>
	<node type="tb_simulate_octrepub_node" pkg="tb_simulate" name="tb_simulate_octrepub_node"/>

</launch>
